backend/src/main/java/botobo/core/quiz/domain/card/FixedCards.java

```diff
@@ -0,0 +1,39 @@
+package botobo.core.quiz.domain.card;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class FixedCards {
+    private static FixedCards fixedCards;
+    private static List<Card> cards;
+
+    private FixedCards(List<Card> cards) {
+        FixedCards.cards = cards;
+    }
+
+    public static FixedCards getInstance() {
+        if (Objects.isNull(fixedCards) && Objects.isNull(cards)) {
+            cards = new ArrayList<>();
+            fixedCards = new FixedCards(cards);
+        }
+        return fixedCards;
+    }
```

Comment:
```suggestion
public class FixedCards {

    private static final FixedCards fixedCards = new FixedCards();

    private final List<Card> cards;

    private FixedCards() {
        cards = new ArrayList<>();
    }

    public static FixedCards getInstance() {
        return fixedCards;
    }
```
이런 방식도 가능할 것 같아서 바꿔봤어요! (테스트도 다 돌려봤습니다!)
코드가 조금 간결해지는데, 조앤의 방식은 필요할 때 객체를 만드는 방식이라 불필요하게 메모리를 차지할 일은 없겠네요.
큰 차이는 아니라 그냥 보시고 조앤이 원하는 방향으로 가면 될 듯! 👍

Reply:
오호 좋은 것 같아서 반영했슴돠! 굳!